packages/untp-test-suite/__tests__/scripts/createConfigFile.test.ts

```diff
@@ -0,0 +1,74 @@
+import fs from 'fs';
+const { createConfigFile } = require('../../scripts/createConfigFile.js');
+
+jest.mock('fs', () => ({
+  writeFile: jest.fn(),
+  readdir: jest.fn(),
+  readdirSync: jest.fn(),
+}));
+
+describe('createConfigFile', () => {
+  it('should call createConfigFile successful', async () => {
+    (fs.readdir as any).mockImplementation((_path, callback) => {
+      callback(null, ['objectEvent', 'productPassport']);
+    });
+
+    (fs.readdirSync as any).mockImplementation(() => ['v0.0.2', 'v0.0.2']);
+
+    (fs.writeFile as any).mockImplementationOnce((_path, _data, callback) => {
+      callback(null);
+    });
+    await createConfigFile();
+
+    expect(fs.writeFile).toHaveBeenCalledTimes(1);
+  });
+
+  it('should show errors when fs.writeFile return error', async () => {
+    (fs.readdir as any).mockImplementation((_path, callback) => {
+      callback(null, ['objectEvent', 'productPassport']);
+    });
+
+    (fs.readdirSync as any).mockImplementation(() => ['v0.0.2', 'v0.0.2']);
+
+    const error = new Error('writeFile error');
+    (fs.writeFile as any).mockImplementationOnce((_path: any, _data: any, callback: (arg0: Error) => any) =>
+      callback(error),
+    );
+
+    try {
+      await createConfigFile();
+    } catch (error) {
+      expect(error).not.toBeNull();
+    }
+  });
+
+  it('should show errors when fs.readdir return error', async () => {
+    const error = new Error('readdir error');
+    (fs.readdir as any).mockImplementation((_path, callback) => {
+      callback(error, null);
+    });
+
+    try {
+      await createConfigFile();
+    } catch (error) {
+      expect(error).not.toBeNull();
+    }
```

Comment:
I think we should expect exactly the error we want to test. In some cases, it might serve a different purpose and throw an error we didn't expect.